src/store/imageBuilderApi.ts

```diff
@@ -1,5 +1,13 @@
 import { emptyImageBuilderApi as api } from "./emptyImageBuilderApi";
+import { fetchBaseQuery } from '@reduxjs/toolkit/query'
+
 const injectedRtkApi = api.injectEndpoints({
+  baseQuery: fetchBaseQuery({paramsSerializer: (params) => {
+      const [multiples, regulars] = Object.entries(params).reduce((acc, [key, value]) => ([{...acc[0], ...Array.isArray(value) && { [key]: value }}, {...acc[1], ...!Array.isArray(value) && { [key]: value }}]), {}),
+          const searchParams = new URLSearchParams(regulars);
+      Object.entries(multiples).forEach(([ key, value ]) => searchParams.append(key, value));
+      return searchParams.toString();
```

Comment:
Can you try to write this in another way ? Or if not, can you add comment about what's happening there and why you need this ? Thanks.